### PR TITLE
Data enrichment test

### PR DESCRIPTION
**Pull request by rockb1017**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -74,7 +74,7 @@ jobs:
             # Disable SSL for HEC new-token
             #sudo /opt/splunk/bin/splunk http-event-collector update -uri https://$CI_SPLUNK_HOST:$CI_SPLUNK_PORT -enable-ssl 0 -auth $CI_SPLUNK_USERNAME:$CI_SPLUNK_PASSWORD
             # Setup Indexes
-            sudo /opt/splunk/bin/splunk add index $CI_INDEX_EVENTS -auth $CI_SPLUNK_USERNAME:$CI_SPLUNK_PASSWORD
+            #sudo /opt/splunk/bin/splunk add index $CI_INDEX_EVENTS -auth $CI_SPLUNK_USERNAME:$CI_SPLUNK_PASSWORD
```
Comment (chaitanyaphalak) [inline .circleci/config.yml:77]: Remove the line if we dont need it anymore :)

File: test/connect_params.json [hunk 2]
```diff
@@ -0,0 +1,105 @@
+[{
```
Comment (chaitanyaphalak) [inline test/connect_params.json:1]: include this file in test/kafka_connect_data_enrichment_tests folder as it specific to that test

File: test/connect_params.py [hunk 3]
```diff
@@ -0,0 +1,118 @@
+import yaml
```
Comment (chaitanyaphalak) [inline test/connect_params.py:1]: include this file in test/kafka_connect_data_enrichment_tests folder as it specific to that test

File: test/connect_params.json (hunk 2 above)
Comment (rockb1017) [reply test/connect_params.json:1]: this file should be deleted.... not using it anymore :)

**Pull request merged**